crypto/comp/c_brotli.c

```diff
@@ -0,0 +1,729 @@
+/*
+ * Copyright 1998-2016 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Is this date a copy/paste?

Reply:
Yes, this file was based on the c_zlib.c file in the same directory. But yeah, it should be updated.

Reply:
Since it's based on c_zlib.c, I'm leaving the start date as 1998.